css/style.css

```diff
@@ -541,3 +699,26 @@ body {
     margin: 64px auto 0;
   }
 }
+
+@media (max-width: 311px) {
+  .test__row{
+    padding: 0 10px;
+  }
+
+  .test__body {
+    font-size: 20px;
+  }
+
+  .test__body_testdrive {
+    font-size: 11px;
+  }
+
+  .btn__test {
+    max-width: 130px;
+    font-size: 11px;
+    max-height: 33px;
+  }
+
+  .test__arrow {
+    margin: 44px auto 0;
+  }}
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5991823)